src/Database/Migrations/2024-01-21-223112_create_radius_tables.php

```diff
@@ -145,6 +145,7 @@ public function up(): void
                 'groupname' => ['type' => 'VARCHAR', 'constraint' => 64, 'default' => ''],
                 'priority'  => ['type' => 'INT', 'constraint' => 11, 'unsigned' => true, 'default' => 1],
             ]);
+            $this->forge->addPrimaryKey('id');
```

Comment:
Hi @jozefrebjak, 
The change you made to the existing migration file introduces a logical issue. 
This is because users who have already executed this migration won't see any effect when they run the migration command again. Previously executed migrations are not re-applied, which means the table won't be updated to include your changes.

A better approach would have been to introduce this change as a new migration file. This way, your updates would be treated as a new migration step and applied to all users consistently.

Alternatively, you could have updated the file name with a newer timestamp, so it gets recognized as a fresh migration.

This approach ensures your changes are always applied and manageable.